platform.md

```diff
@@ -459,6 +460,9 @@ Usage:
         - `io.buildpacks.lifecycle.metadata`: see [lifecycle metadata label (JSON)](#lifecycle-metadata-label-(json))
         - `io.buildpacks.project.metadata`: the value of which SHALL be the json representation `<project-metadata>`
         - `io.buildpacks.build.metadata`: see [build metadata (JSON)](#build-metadata-label-(json))
+- To ensure [build reproducibility](#build-reproducibility), the lifecycle:
+    - SHOULD set the modification time of all files in newly created layers to a constant value
```

Comment:
does the default belong in the spec?

Reply:
Personally I don't think so. We choose (00:00:01, January 1st, 1980 in UTC for compatibility with zlib) but I think a hypothetical other lifecycle that used the unix epoch should still be compliant.

Reply:
Agreed.  The consistency must be between multiple runs using the same implementation, not between different implementations.